RecoHI/HiTracking/python/HIPixelVerticesPreSplitting_cff.py

```diff
@@ -38,21 +38,23 @@
   adaptiveVertexCollection = cms.InputTag("hiBestAdaptiveVertexPreSplitting"),
   medianVertexCollection = cms.InputTag("hiPixelMedianVertexPreSplitting")
 )
-bestHiVertexPreSplitting = cms.Sequence( hiBestAdaptiveVertexPreSplitting * hiSelectedVertexPreSplitting )
+bestHiVertexPreSplittingTask = cms.Task( hiBestAdaptiveVertexPreSplitting , hiSelectedVertexPreSplitting )
+bestHiVertexPreSplitting = cms.Sequence(bestHiVertexPreSplittingTask)
```

Comment:
This was only needed here internally: sequence not needed any more now, please remove